### ISSUE TITLE
[Feature] Add support for equation numbering

### ISSUE THREAD

**Issue by SteelWagstaff** [pressbooks/pb-mathjax#32]
MathJax allows users to include automatic numbering for equations. See https://docs.mathjax.org/en/v2.7-latest/tex.html#automatic-equation-numbering + http://docs.mathjax.org/en/latest/input/tex/eqnumbers.html. We do not support this feature in Pressbooks currently. 

Turning it on in webbooks seems fairly straightforward, but we will want to ensure this works properly in ebook and PDF exports. #23 is related

**Comment by pb-amy** [comment]
[November 2021] This was reported / requested by Rebus: https://pressbooks.zendesk.com/agent/tickets/7703

**Comment by apurvaashok** [comment]
Yes, we're hoping that this can be implemented across the board soon. This is proving to be an important feature in many STEM texts we are supporting. They rely on MathJax to render accessible formulae and need to reference equations at different points throughout the texts — numbering/indexing/labelling is a surefire way to point to equations across all the formats (rather than asking users to look at the equations 'above' or 'below' another element in the book). It'll make for a much easier workflow and aesthetic, accessible book!

**Comment by apurvaashok** [comment]
Hey team - just checking in on whether this issue is being addressed in any current sprints/planned for the near future?

**Comment by tw77** [comment]
[May 2022] This has also been requested by Virginia Tech: https://pressbooks.zendesk.com/agent/tickets/10583

**Comment by tw77** [comment]
[July 2022] New thread on this topic: https://pressbooks.community/t/use-of-latex-for-equations/

**Comment by SteelWagstaff** [comment]
@os @tw77 this is one that is probably worth testing on our next release candidate (especially in exports). If it doesn't work well there, we should probably refine and bump this issue so it could be included in the next set of improvements.

**Comment by acheveldave** [comment]
Hi there,

I see that numbering for LaTeX equations rendered using MathJax does work in the web book now, but it doesn't work in PDF exports. Is this feature still being explored? I had a request for this feature from a user of BCcampus' open-source self-publishing instance of Pressbooks, pressbooks.bccampus.ca.

**Comment by arzola** [comment]
This issue has been automatically closed as part of an organization-wide backlog review.

**Reason:** This issue was created 5y 4m ago with no updates in the last 2m.

If this is still relevant, please feel free to reopen it with updated context. We appreciate your contributions and want to ensure our issue tracker stays actionable.

**Learn more:** https://pressbooks.community/t/announcement-automatic-closure-of-old-github-tickets/4542
